### DISCUSSION TITLE
Add stack/deck mode for column

### DISCUSSION THREAD

**Discussion by ZabejAga** [YaLTeR/niri#835] [Ideas]
Here is an idea:
in case 2+ windows in column be able to switch mode to deck (dwm, qtile) aka stack (i3) where the top (or last focused window) are visible at full height. Its better to have separate color/border thicknes for such deck/stack. Keyboard shortcut with arrow up/down should  rise windows under top window. 
useful for terminals, file managers, chat programs, ie arranged by type.

**Comment by YaLTeR** [comment]
Hey, I guess like in the existing discussion: https://github.com/YaLTeR/niri/discussions/698

**Reply by ZabejAga** [reply]
looks like. 
soryan )
